pkg/food/food.go

```diff
@@ -0,0 +1,35 @@
+package food
+
+import (
+	"encoding/json"
+)
+
+const (
+	Drinks = iota
+	Foods
+)
+
+type Food struct {
+	Id          uint64
+	UserId      uint64
+	Type        uint8
+	Name        string
+	PortionSize float32
+}
+
+func (f *Food) String() string {
+	mf, err := json.Marshal(f)
+	if err != nil {
+		panic(err.Error())
+	}
+	return string(mf)
+}
+
+func CreateFood(foodInfo []byte) (f *Food) {
```

Comment:
```suggestion
func CreateFood(foodInfo []byte) (*Food) {
```
лучше не называть переменные если они не используются, особенно такая путаница бывает с err, тут простое правило если переменные названы - значит используются и надо смотреть где)

Reply:
да, сам же забыл пока писал...убрал